### ISSUE TITLE
Add CNI Chart To Helm Hub

### ISSUE THREAD

**Issue by ihcsim** [linkerd/linkerd2#3802]
Add the Linkerd CNI chart to Helm Hub. See the Helm Hub [docs](https://github.com/helm/hub/blob/master/Repositories.md) for more information.

Depends on #3801.
See #3367 for previous work.

**Comment by zaharidichev** [comment]
@alpeb As far as I take it this is blocked on releasing 2.7 as there is no CNI chart in the stable channel now because it has not yet been built by the CI, correct?

**Comment by alpeb** [comment]
@zaharidichev It appears there's nothing to do here. Only a config at the repo level `(https://helm.linkerd.io/(edge|stable)` is needed, which we already did when the linkerd2 chart was added to the Helm Hub. And after last week's edge the linkerd2-cni chart was automatically picked up:
https://hub.helm.sh/charts/linkerd2-edge/linkerd2-cni
So when stable-2.7.0 gets released I expect it well get picked automatically as well.
